cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -114,6 +115,96 @@ def test_xlint_fails(self):
         err_cnt = perform_xlint(TEST_DATA_DIR, ['toy'])
         self.assertGreater(err_cnt, 0)
 
+    def test_invalid_asset_name_overwrite(self):
```

Comment:
Can we specify asset name explicitly in test name or in doc strings to make it more readable.